modules/core/src/main/kotlin/com/metalbear/mirrord/MirrordApi.kt

```diff
@@ -607,6 +656,9 @@ private abstract class MirrordCliTask<T>(private val cli: String, private val co
                 environment.put("MIRRORD_TARGET_NAMESPACE", it)
             }
 
+            // for config explanation to be printed out
+            environment["MIRRORD_EXT_PRINT_CONFIG"] = "TRUE"
```

Comment:
Is this equivalent to the `environment.put`?

Reply:
yes, lint recommended I do it (not sure why we don't do it elsewere..

Reply:
Alright, just making sure.